games/s.yaml

```diff
@@ -800,6 +908,8 @@
     license: [GPL3]
 
 - name: [Similar to Natural Selection, Natural Selection (video game)]
```

Comment:
This should be in the n.yaml file I think; not sure why "Similar to" is necessary